wca-integration/src/main/java/eu/itesla_project/wca/WCAUtils.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.wca;
+
+import java.nio.file.Path;
+import java.util.Objects;
+import java.util.Properties;
+
+import eu.itesla_project.iidm.datasource.DataSource;
+import eu.itesla_project.iidm.datasource.GzFileDataSource;
+import eu.itesla_project.iidm.export.Exporters;
+import eu.itesla_project.iidm.network.Network;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.it>
+ */
+public final class WCAUtils {
+
+    private WCAUtils() {
+    }
```

Comment:
I think that old utility classes should be converted to utility interfaces with static methods. In this specific case, there is no reason to create an instance (no internal data), so the exportState method is static but you have to declare this class as final and add a private constructor.
This pattern is equivalent to an interface with a single static method. This interface can be private package.

This is a suggestion: if you think your implementation is better, just keep it.